.github/workflows/cmd.yml

```diff
@@ -0,0 +1,417 @@
+name: Command
+
+on:
+  issue_comment: # listen for comments on issues
+    types: [ created ]
+
+permissions: # allow the action to comment on the PR
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: read
+
+jobs:
+  is-org-member:
+    if: startsWith(github.event.comment.body, '/cmd')
+    runs-on: ubuntu-latest
+    outputs:
+      member: ${{ steps.is-member.outputs.result }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
It looks like it's not needed to checkout the repo if the job checks only org membership